### PR TITLE
(chores) camel-core: test isolation + parallelization fixes

### PR DESCRIPTION
**Pull request by orpiske**

### CODE REVIEW

**Comment by orpiske** [comment]
These are particularly problematic on s390x.

**Review by oscerd — APPROVED** [review]

File: core/camel-core/src/test/java/org/apache/camel/impl/ScheduledPollConsumerBackoffTest.java [hunk 1]
```diff
@@ -22,10 +22,12 @@
 import org.apache.camel.spi.PollingConsumerPollStrategy;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.condition.DisabledIfSystemProperty;
+import org.junit.jupiter.api.parallel.Isolated;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
 @DisabledIfSystemProperty(named = "ci.env.name", matches = "github.com", disabledReason = "Flaky on Github CI")
```
Comment (essobedo) [inline core/camel-core/src/test/java/org/apache/camel/impl/ScheduledPollConsumerBackoffTest.java:29]: Can it be removed?

Comment (orpiske) [reply core/camel-core/src/test/java/org/apache/camel/impl/ScheduledPollConsumerBackoffTest.java:29]: I don't think so, but it won't hurt to test. Let's give it a try.

**Comment by orpiske** [comment]
> 🚫 There are (likely) no components to be tested in this PR

On a side note: we need to implement a tester for the core code: https://issues.apache.org/jira/browse/CAMEL-19427

**Review by davsclaus — APPROVED** [review]

**Pull request merged**